rfcs/2021-01-05-typography-scaling.md

```diff
@@ -0,0 +1,223 @@
+
+#### **On Desktop**
+<img width="600" src="./images/typography/dashboard-example-desktop.png" />
+
+---
+
+#### **On Mobile**
+<img width="300" src="./images/typography/dashboard-example-mobile.png" />
+
+<br>
+
+# Motivation
+### **Why do we need a type scale?**
+We need our users to access our products anywhere, anytime. To make sure they can do so, we need to be present on almost every device possible especially on portables (mobiles & tablets). Every device offers a different experience and we need to make sure that we are flexible in that case. One of the major aspects of that experience is **readability** and therefore we are focusing on **typography scaling** across multiple devices.
+
+### **Use Cases**
+There can be several use-cases when a user wants to access the dashboard on multiple devices. Few of those can be,
+- **On Mobiles** (user can check for settlement numbers while commuting/traveling in a metro/bus)
+- **On Tablets** (user can capture a payment while on-the-go to their home back from the office)
+- **On Desktops** (user can request for access to the international payment via the dashboard)
+
+Or in-general a user can visit the company's marketing/landing page and consume information about us and our offerings.
+
+### **Examples from the current system**
+Here are a few examples in the current system.
+- **Dashboard Route - Desktop vs. Mobile:**
+![Dashboard Route: Desktop vs. Mobile](./images/typography/dashboard-route-mobile-desktop-comparison.png)
+
+- **Dashboard Invoices - Desktop vs. Mobile:**
+![Dashboard Invoices: Desktop vs. Mobile](./images/typography/dashboard-invoices-mobile-desktop-comparison.png)
+
+- **Dashboard Invoices (Mobile) - Font Ratios:**
+![Dashboard Invoices (Mobile): Font Ratios](./images/typography/dashboard-invoices-mobile-problems.png)
+
+In the above examples, you can see there is no scaling across different devices/screens. Which is not considered as a good experience as we are compromising on the readability of things on different devices.
+### **Expected Outcome**
+A typography system that is more readable/consumable across different devices (mobiles/tablets/desktops).
+
+# Detailed Design
+
+### **Introduction**
+We are following a [**Major Second**](https://type-scale.com/?size=14&scale=1.125&text=A%20Visual%20Type%20Scale&font=Lato&fontweight=400&bodyfont=Poppins&bodyfontweight=400&lineheight=1.65&backgroundcolor=white&fontcolor=%23333&preview=false) modular scale (`1.125`) for our typography system on larger screens (desktops/tablets). And [**Custom**](https://type-scale.com/?size=14&scale=1.125&text=A%20Visual%20Type%20Scale&font=Lato&fontweight=400&bodyfont=Poppins&bodyfontweight=400&lineheight=1.65&backgroundcolor=white&fontcolor=%23333&preview=false) modular scale (`1.1`) for our typography system on smaller screens (mobiles). Simply put, the font-sizes will be derived in multiples of 1.125 & 1.1 for large screens and small screens respectively.
```

Comment:
why custom scale for mobile? aren't the other standard scale working for us?

Reply:
No, the other scales were not working, the gap between them was too high. `1.1` was working well for our use case.
_Disclaimer If we change the font in the future, this might change._